crates/wasmtime/src/runtime/component/types.rs

```diff
@@ -799,24 +799,30 @@ impl Module {
     }
 
     /// Iterates over imports of the module
-    pub fn imports(&self) -> impl ExactSizeIterator<Item = ((&str, &str), ExternType)> {
+    pub fn imports<'a>(
+        &'a self,
+        engine: &'a Engine,
```

Comment:
I think this is a bit unfortunate and ideally we'd wrap this via some other means, for example storing a `Component` in `Handle<T>` or an `Engine` in there or something like that. That being said this isn't the end of the world so I think it's reasonable to keep it around for now and see if we can figure out something better in the future.